plugins/hiscore/init.lua

```diff
@@ -172,6 +197,7 @@ function hiscore.startplugin()
 	  if not output then
 		-- attempt to create the directory, and try again
 		lfs.mkdir( hiscore_path );
+		lfs.mkdir( hiscore_path .. '/' .. emu.romname() );
```

Comment:
Why are you always creating this path when it’s not going to be used unless stuff is found for the current software?

Reply:
ok, i'll fix it with this check:
````
		if manager:machine().images["cart"] or manager:machine().images["cdrom"] ~= "" then
			lfs.mkdir( hiscore_path .. '/' .. emu.romname() );
		end
````